lib/project_types/script/forms/script_form.rb

```diff
@@ -61,6 +71,15 @@ def ask_shop_domain(organization, message: ctx.message('script.forms.script_form
           CLI::UI::Prompt.ask(message, options: organization["stores"].map { |s| s["shopDomain"] })
         end
       end
+
+      def write_env(org, api_key, secret_key, shop)
+        ShopifyCli::Resources::EnvFile.new(
```

Comment:
Looks like ARGO puts this logic within the ExtensionProject. I think it would make sense to put this in our ScriptProject